extensions/standard-processors/processors/PutTCP.cpp

```diff
@@ -215,7 +215,7 @@ asio::awaitable<std::error_code> ConnectionHandler<SocketType>::establishNewConn
       continue;
     }
     if (max_size_of_socket_send_buffer_)
-      socket.lowest_layer().set_option(TcpSocket::send_buffer_size(*max_size_of_socket_send_buffer_));
+      socket.lowest_layer().set_option(TcpSocket::send_buffer_size(*max_size_of_socket_send_buffer_));  // NOLINT(google-readability-casting)
```

Comment:
What was the problem here?

Reply:
Same false positive, clang-tidy recognizing `TcpSocket::send_buffer_size()` call as a C-style cast

Reply:
I think casts are pretty well covered by other checks. I would just disable this one due to the false positives, instead of littering the code with NOLINTs.

Reply:
Removed in b1d4a35c9a2021f2bd8d0fa750e2f2971f635cae